src/main/java/com/capstone/producer/kafka/KafkaProducer.java

```diff
@@ -21,13 +22,14 @@
 @Service
 public class KafkaProducer {
 
-    private final static String TOPIC_NAME = "FlightData";
-
-    @Value("${kafka.host}")
-    private final static String BOOTSTRAP_SERVER = "kafka:9092";
+    private static String BOOTSTRAP_SERVER;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaProducer.class);
 
+    @Autowired
+    public KafkaProducer(@Value("${kafka.host}") String BOOTSTRAP_SERVER) {
```

Comment:
Could you replace the variable name with one that is lowercase. Sonar lint would flag this as a code smell

Reply:
Done, although you should probably look closer at how I had to modify the bootstrap server for kafka, it for some reason wasn't working the way you set it after creating the environment variables. Not sure why... I might have just been doing it wrong.

Reply:
I actually reverted it back to how you had it, we just should look at it further some point.